MapboxSearch/base/src/main/java/com/mapbox/search/base/engine/TwoStepsRequestCallbackWrapper.kt

```diff
@@ -126,6 +126,14 @@ class TwoStepsRequestCallbackWrapper(
                     searchRequestTask.markExecutedAndRunOnCallback(callbackExecutor) {
                         (this as BaseSearchSelectionCallback).onResults(suggestion, results, responseInfo)
                     }
+                } else if (suggestion != null && responseResult.isEmpty()) {
+                    val error = IOException(
```

Comment:
I don't know why this is an IOException, but I see it was used in older code as well.

Reply:
This is supposed to be apart of subsequent pull request. I need to revert this code.